openfisca_france/model/prestations/minima_sociaux/ppa.py

```diff
@@ -198,6 +202,12 @@ class ppa_ressources_hors_activite_individu(Variable):
     entity = Individu
     label = u"Revenu hors activité pris en compte pour la PPA (Individu) pour un mois"
     definition_period = MONTH
+    reference = [
+        u"Article L842-4 du code de la sécurité sociale",
```

Comment:
```suggestion
        # Article L842-4 du code de la sécurité sociale
```

Sinon, sur legislation explorer, on ajoute de liens morts :

<img width="808" alt="capture d ecran 2018-12-28 a 15 38 42" src="https://user-images.githubusercontent.com/329236/50518590-b21bc200-0ab6-11e9-99f4-2a3a41cdcd3f.png">